lib/screens/play/play.dart

```diff
@@ -34,6 +40,15 @@ class _PlayPageState extends State<PlayPage> {
       Resource.DOUBLOON,
       Resource.DOUBLOON,
     ]);
+
+    gameState.nextCard();
+  }
+
+  List<GameCard.Card> _getInitialDeck() {
+    return [
+      PlunderAWreck(),
+      ARivalShip()
+    ];
   }
```

Comment:
Obviously just temporary so it's fine, but would be keen for wherever we refactor this in future to live outside of UI code.

Reply:
Added a //Todo for this